source/_components/device_tracker.tile.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Tile"
+description: "Instructions how to use Tile to track devices in Home Assistant."
+date: 2017-11-08 20:40:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tile.png
+ha_release: 0.58
+ha_category: Presence Detection
+ha_iot_class: "Cloud Polling"
+---
+
+The `tile` platform allows Home Assistant to utilize [Tile® Bluetooth trackers](https://www.thetileapp.com).
+The official Tile mobile app handles the actual tracking of Tile devices using
+the mobile device's Bluetooh and GPS.
+
+To integrate Tile into Home Assistant, add the following section to your
+`configuration.yaml` file:
+
+```yaml
+device_tracker:
+  - platform: tile
+    email: email@address.com
+    password: MY_PASSWORD_123
+    monitored_variables:
+      - TILE
+      - PHONE
+```
+
+Configuration variables:
```

Comment:
Use configuration tag:
https://home-assistant.io/developers/documentation/standards/#component-and-platform-pages